<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pharmacy Scrapper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralwidget_verticalLayout">
    <item>
     <widget class="QFrame" name="logo_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="logo_frame_verticalLayout">
       <item>
        <widget class="QLabel" name="logo_label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../icons/logo.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="select_pharmacies_groupBox">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: QLinearGradient( x1: 0, y1: 0,
                                                        x2: 1, y2: 0,
                                                    stop: 0 #29b0a7,
                                                    stop: 1 #8de4df );
	color: #FFFFFF;
	border: none;
	padding: 10px;
	border-radius: 10px;
}
QPushButton:pressed {
	background-color: #4fd6cd;
}
QPushButton:checked {
	background-color: #bdefec;
}</string>
      </property>
      <property name="title">
       <string>Select pharmacies to scrap</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="select_pharmacies_groupBox_horizontalLayout">
       <item>
        <widget class="QPushButton" name="pharmacy_button_1">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Aptelia</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pharmacy_button_2">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cefarm24</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pharmacy_button_3">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Gemini</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pharmacy_button_4">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Melissa</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pharmacy_button_5">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ziko</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="product_name_groupBox">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Enter products names</string>
      </property>
      <layout class="QVBoxLayout" name="product_name_groupBox_verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>762</width>
            <height>112</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="scrollAreaWidgetLayout">
           <item>
            <widget class="QFrame" name="product_frame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="product_frame_horizontalLayout">
              <item>
               <widget class="QLabel" name="product_label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Product 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextBrowser" name="product_name_textBrowser">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="documentTitle">
                 <string/>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>Enter the product name</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="buttons_frame">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="buttons_frame_horizontalLayout">
          <item>
           <widget class="QPushButton" name="add_product_pushButton">
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: QLinearGradient( x1: 0, y1: 0,
                                                        x2: 1, y2: 0,
                                                    stop: 0 #534ad4,
                                                    stop: 1 #ada9eb );
	color: #FFFFFF;
	border: none;
	padding: 10px;
	border-radius: 10px;
}
QPushButton:pressed {
	background-color: #807ae0;
}</string>
            </property>
            <property name="text">
             <string>Add next product</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_product_pushButton">
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: QLinearGradient( x1: 0, y1: 0,
                                                        x2: 1, y2: 0,
                                                    stop: 0 #8749c2,
                                                    stop: 1 #b792db );
	color: #FFFFFF;
	border: none;
	padding: 10px;
	border-radius: 10px;
}
QPushButton:pressed {
	background-color: #9058c7;
}</string>
            </property>
            <property name="text">
             <string>Remove last product</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="run_button_frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="run_button_frame_horizontalLayout">
       <item>
        <spacer name="run_button_frame_horizontalSpacer_left">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="run_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: QLinearGradient( x1: 0, y1: 0,
                                                        x2: 1, y2: 0,
                                                    stop: 0 #ff5252,
                                                    stop: 1 #ffb4b4 );
	color: #FFFFFF;
	border: none;
	padding: 10px;
	border-radius: 10px;
}
QPushButton:pressed {
	background-color: #ff7979;
}</string>
         </property>
         <property name="text">
          <string>RUN</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="run_button_frame_horizontalSpacer_right">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
